device-types/Quantum/QXS-424.yaml

```diff
@@ -0,0 +1,45 @@
+---
+manufacturer: Quantum
+model: QXS-424
+slug: quantum-qxs-424
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
+subdevice_role: parent
+power-ports:
+  - name: '1'
+    label: PSU Slot 1
+    type: iec-60320-c14
+    position: '1'
+  - name: '2'
+    label: PSU Slot 2
+    type: iec-60320-c14
+    position: '2'
+device-bays:
+  - name: A
+    label: Controller Slot A
+  - name: B
+    label: Controller Slot B
+interfaces:
```

Comment:
As these ports are on the controller rather than the chassis they shouldn't be modeled on the chassis device

Reply:
No, you need to create a seperate device for the controller that has the interfaces defined.